akka-stream/src/main/boilerplate/akka/japi/tuple/Tuple.scala.template

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.japi.tuple
+
+[2..22#/**
+ * Tuple container. Used to create tuples in Java.
+ */
+@SerialVersionUID(##1L)
+case class Tuple1[[#T1#]] ([#val t1: T1#]) extends scala.AnyRef with scala.Product with scala.Serializable{
+   val toScala = ([#t1#])
+}#
```

Comment:
Most of our JavaAPI has "create" methods, let's add one here as well so it's consistent:

```scala
object Tuple1 {
  def create[[#T1#]]([#t1: T1#]) = new Tuple1<>([#t1#])
```

I personally like guava's `Pair.of` but let's stick to our `create` style.